packages/action-group/test/action-group.test.ts

```diff
@@ -121,6 +122,34 @@ async function multipleSelectedActionGroup(
 }
 
 describe('ActionGroup', () => {
+    it('throws an error if slotElement is null', async () => {
+        // To verify that this test is not evergreen, you can temporarily disable the safeguard
+        // clause in `manageButtons` by commenting out the following lines:
+        // if (!this.slotElement) { return; }
+        const el = await fixture<ActionGroup>(html`
+            <sp-action-group>
+                <sp-action-button value="first">First</sp-action-button>
+                <sp-action-button value="second">Second</sp-action-button>
+            </sp-action-group>
+        `);
+
+        // Stub the slotElement getter to return null
+        const slotElementStub = sinon.stub(el, 'slotElement').get(() => null);
+
+        // Call manageButtons and expect a TypeError
+        expect(() => Reflect.get(el, 'manageButtons').call(el)).not.to.throw(
```

Comment:
What is Reflect?

Reply:
You can check out more details [here on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect/get), but basically, I’m using `Reflect.get` as a hack to access the private `manageButtons` method on `el` in the test.
I tried a few different approaches to directly test `manageButtons`, including adding a subclass just for testing to expose these private methods (a pattern we could actually explore more if we want to start getting more isolated unit tests for private methods). Reflect.get, however, sounded like a good compromise solution.
I am happy to try out other ideas though!